### ISSUE TITLE
An easier way to "jump" forward/backward in the network feed

### ISSUE THREAD

**Issue by DetroitLibertyPenguin** [friendica/friendica#11434]
### Is the feature request related to a problem? Please describe.

I frequently see the same posts over and over again as I check through out the day, but I suspect there's a lot from last night, last week, etc. that I haven't seen yet. 

### Describe the feature you'd like

I'd like a button or something that says "jump 1 hour, Jump 6 hours, jump 12 hours, jump 1 day, etc." 

### Describe alternatives you've considered

I can edit the URL directly like https://libranet.de/network?max_id=2022-04-22%2004:40:13 but I don't really think like that, I don't remember what times' I've looked at and where I should jupm to next, though its easier than haveing to scroll to the bottom and click NEXT, particularly on mobile browser. 

### Additional context

**Comment by MrPetovan** [comment]
Have you tried the "Latest Posts" order instead of the "Latest Activity"? This way the order of the conversations in your timeline stays consistent over time. The drawback is that you have to subscribe to the conversations you want to be notified about for new comments, but this is done automatically when you add a like or comment in the conversation.

**Comment by DetroitLibertyPenguin** [comment]
Yeah, I've tried both. Most of my concern is about reading stuff for the
first time that was posted while I was sleeping or otherwise out having an
actual life rather than staring at my phone/computer ;-)




Jim Fulner
22100 Armada Ridge Rd.,
Armada, MI 48005-3204
(P) 248-971-0259
(M) 248-765-7458
(H) 586-473-7007

Telephony eliminates confusion

**Comment by DetroitLibertyPenguin** [comment]
I usually have it set to Latests posts.
